test/Models/TerriaSpec.ts

```diff
@@ -36,22 +36,18 @@ import { buildShareLink } from "../../lib/ReactViews/Map/Panels/SharePanel/Build
 import SimpleCatalogItem from "../Helpers/SimpleCatalogItem";
 import { defaultBaseMaps } from "../../lib/Models/BaseMaps/defaultBaseMaps";
 
-const mapConfigBasicJson = require("../../wwwroot/test/Magda/map-config-basic.json");
-const mapConfigBasicString = JSON.stringify(mapConfigBasicJson);
-
-const mapConfigV7Json = require("../../wwwroot/test/Magda/map-config-v7.json");
-const mapConfigV7String = JSON.stringify(mapConfigV7Json);
-
-const mapConfigInlineInitJson = require("../../wwwroot/test/Magda/map-config-inline-init.json");
-const mapConfigInlineInitString = JSON.stringify(mapConfigInlineInitJson);
-
-const mapConfigDereferencedJson = require("../../wwwroot/test/Magda/map-config-dereferenced.json");
-const mapConfigDereferencedString = JSON.stringify(mapConfigDereferencedJson);
-
-const mapConfigDereferencedNewJson = require("../../wwwroot/test/Magda/map-config-dereferenced-new.json");
-const mapConfigDereferencedNewString = JSON.stringify(
-  mapConfigDereferencedNewJson
-);
+import mapConfigBasicJson from "../../wwwroot/test/Magda/map-config-basic.json";
+import mapConfigV7Json from "../../wwwroot/test/Magda/map-config-v7.json";
+import mapConfigInlineInitJson from "../../wwwroot/test/Magda/map-config-inline-init.json";
```

Comment:
When I run `yarn gulp build` on this branch, I get this warning just above the prompt after the build completes:
```
WARNING in ./node_modules/terriajs-cesium/Source/Core/buildModuleUrl.js 50:37-48
Critical dependency: Accessing import.meta directly is unsupported (only property access or destructuring is supported)
 @ ./test/Models/TerriaSpec.ts 41:0-72 113:13-27 118:13-27
```